data/json/monsters/zed-animal.json

```diff
@@ -376,5 +376,51 @@
       "REVIVES",
       "FILTHY"
     ]
+  },
+  {
+    "id": "mon_zpider_mass",
+    "type": "MONSTER",
+    "name": { "str": "mass of zombie spiders" },
+    "description": "Dozens, hundreds of spiders piling up high, each slowly oozing sticky green pus, struggling to keep the fetid mass together and moving.",
```

Comment:
23 kg of spider is much much more than hundreds of them ^^"

Reply:
Yeah, can this be changed to something like "thousands, maybe even millions", the heaviest spider irl is ~170g, so its thousands at minimum really.